en/sim_gazebo_gz/worlds.md

```diff
@@ -0,0 +1,80 @@
+# Gazebo Worlds
+
+This topic provides imagery/information about the [Gazebo](../sim_gazebo_gz/index.md) worlds supported by PX4.
+
+The [default world](#default) is spawned by default, though this may be overridden by a [model specific world](#model_specific_worlds).
+Developers can also manually specify the world to load: [Gazebo > Specify World](../sim_gazebo_gz/index.md#specify-world) (or [Gazebo Models Repository](../sim_gazebo_gz/gazebo_models.md#gazebo-models-repository-px4-gazebo-models)).
+
+The source code for supported worlds can be found in the [Gazebo Models Repository](../sim_gazebo_gz/gazebo_models.md#gazebo-models-repository-px4-gazebo-models) on GitHub here: [PX4/PX4-gazebo-models/tree/main/worlds](https://github.com/PX4/PX4-gazebo-models/tree/main/worlds).
+
+## Empty (Default) {#default}
+
+Empty world (a grey plane).
+This is used by default.
+
+[PX4-gazebo-models/main/worlds/default.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/default.sdf)
+
+![screenshot of default world](../../assets/simulation/gazebo/worlds/default.png)
+
+## Aruco
+
+TBD: What's it for? What vehicles should it be used with. Is it used by any vehicles by default?
+
+[PX4-gazebo-models/main/worlds/aruco.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/aruco.sdf)
+
+![screenshot of Aruco world](../../assets/simulation/gazebo/worlds/aruco.png)
+
+## Baylands
+
+Baylands world surrounded by water.
+
+[PX4-gazebo-models/main/worlds/bayland.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/baylands.sdf)
+
+![screenshot of Baylands world](../../assets/simulation/gazebo/worlds/baylands.png)
+
+## Lawn
+
+TBD: What's it for? What vehicles should it be used with. Is it used by any vehicles by default?
+
+[PX4-gazebo-models/main/worlds/lawn.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/lawn.sdf)
+
+![screenshot of lawn world](../../assets/simulation/gazebo/worlds/lawn.png)
+
+## Rover
+
+TBD: What's it for? What vehicles should it be used with. Is it used by any vehicles by default?
+
+[PX4-gazebo-models/main/worlds/rover.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/rover.sdf)
+
+![screenshot of rover world](../../assets/simulation/gazebo/worlds/rover.png)
+
+## Walls
+
+World for testing collision prevention.
+
+[PX4-gazebo-models/main/worlds/walls.sdf](https://github.com/PX4/PX4-gazebo-models/blob/main/worlds/walls.sdf)
+
+![screenshot of walls world](../../assets/simulation/gazebo/worlds/walls.png)
+
+## Windy
+
+[Empty world](#default) with wind enabled.
```

Comment:
What's it for? What vehicles should it be used with. Is it used by any vehicles by default?

Can we say anything about the wind here - i.e how windy? Anything configurable? Is he wind variable or consistent or noisy. Anything developers might find relevant?

Reply:
@frede791 Can you advise ^^^ when you might use this/what you'd like to show? Any configuration? I'm going to merge this PR because this shouldn't block, but I know people will want to know if the world is suitable before they try it.